src/LineBot/LineBot.cs

```diff
@@ -222,9 +222,9 @@ public async Task<IEnumerable<ILineEvent>> GetEvents(HttpRequest request)
             var eventCollection = JsonConvert.DeserializeObject<LineEventCollection>(jsonContent);
 
             if (eventCollection == null || eventCollection.Events == null)
-                return Enumerable.Empty<ILineEvent>();
+                return LineEventCollection.Empty();
```

Comment:
I think we should only return `LineEventCollection.Empty` when `eventCollection` is `null`. And set the `Events` property to `new List<LineEvent>()` when `eventCollection` is not `null`. Otherwise the user of the API will not get the value of `Destination`. Probably also need a new test for this.